src/backend/commands/functioncmds.c

```diff
@@ -2447,7 +2447,11 @@ ExecuteCallStmt(CallStmt *stmt, ParamListInfo params, bool atomic, DestReceiver
 		i++;
 	}
 
-	/*
+	/* Get rid of temporary snapshot for arguments, if we made one */
```

Comment:
I don't think this is what we want here. I'm pretty sure the original community code had this line. If that's the case, the fact that this commit is adding it back in means that the community actually removed this bit in 14.x.

Reply:
Community code has this line.
https://github.com/postgres/postgres/blob/REL_14_2/src/backend/commands/functioncmds.c